.github/workflows/deploy-website.yml

```diff
@@ -0,0 +1,28 @@
+name: "Deploy website"
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy_website:
+    name: "Deploy website"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+          cache: 'pip'
+
+      - name: Install necessary tools
+        run: pip install -r requirements.txt
+
+      - name: Copy Javadoc
+        run: cp -r javadoc/ docs/javadoc/
```

Comment:
I didn't move the `javadoc` folder to the `docs` folder to not make this PR even bigger.
But I'll submit an other PR once/if this one is merged, to move the `javadoc` folder, and remove this step.